cloud/google/cmd/gce-machine-controller/Makefile

```diff
@@ -18,7 +18,7 @@ GCR_BUCKET = k8s-cluster-api
 PREFIX = gcr.io/$(GCR_BUCKET)
 DEV_PREFIX ?= gcr.io/$(shell gcloud config get-value project)
 NAME = gce-machine-controller
```

Comment:
(sanity check)
Do the controller manager and/or apiserver need to be updated too? I recall previous changes to provider config needing to touch more than just the gce-controller.

Reply:
This was probably the provider config field itself changing, not the format of the data stored in it.

Reply:
Could be. So long as the cluster spins up and you can create and delete machines without issue then you are good.